backend/app/resources/auth_resources.py

```diff
@@ -46,5 +49,5 @@ def post(self):
             stored_hash = base64.b64decode(user.password_hash.encode('utf-8'))
             if checkpw(password_bytes, stored_hash):
                 access_token = create_access_token(identity=user.id)
-                return {'access_token': access_token, 'msg': 'Registered Successfully'}, 200
+                return {'access_token': access_token, 'message': 'Logged In Successfully'}, 200
```

Comment:
It’s more effective to define a general API response using an abstract class and encapsulate all responses within it.